crates/cadenza-eval/src/eval.rs

```diff
@@ -147,40 +117,39 @@ fn eval_apply(
     // Check if the receiver is an identifier that names a macro
     if let Expr::Ident(ref ident) = receiver_expr {
         let text = ident.syntax().text().to_string();
```

Comment:
We shouldn't be allocating a string just to intern it! That's the whole point of interning is to avoid having so many active allocations everywhere.